lib/stencil-init.js

```diff
@@ -70,38 +85,46 @@ class StencilInit {
 
     /**
      * @param {{port: (number), normalStoreUrl: (string), accessToken: (string)}} stencilConfig
-     * @param {{port: (number), url: (string), token: (string)}} cliOptions
      * @returns {{port: (number), normalStoreUrl: (string), accessToken: (string)}}
      */
-    getDefaultAnswers (stencilConfig, cliOptions) {
+    getDefaultAnswers (stencilConfig) {
         return {
-            normalStoreUrl: cliOptions.url || stencilConfig.normalStoreUrl,
-            accessToken: cliOptions.token || stencilConfig.accessToken,
-            port: cliOptions.port || stencilConfig.port || this.serverConfig.get('/server/port'),
+            normalStoreUrl: stencilConfig.normalStoreUrl,
+            accessToken: stencilConfig.accessToken,
+            port: stencilConfig.port || this.serverConfig.get('/server/port'),
         };
     }
 
     /**
      * @param {{port: (number), normalStoreUrl: (string), accessToken: (string)}} defaultAnswers
-     * @returns {Promise<object>}
+     * @param {{port: (number), normalStoreUrl: (string), accessToken: (string)}} cliConfig
+     * @returns {{port: (number), normalStoreUrl: (string), accessToken: (string)}}
```

Comment:
Seems like we no longer return the object, but `object[] | null` instead.

Reply:
Actually I would return an empty array rather than null in case of no questions for the simplicity of the types (and then use `questions.lenght ? await this.askQuestions(questions) : {}`), but it is optional since not a mistake, but just a personal preference.